stdlib/public/runtime/Metadata.cpp

```diff
@@ -4430,6 +4430,12 @@ ExtendedExistentialTypeCacheEntry::getOrCreateVWT(Key key) {
   swift_unreachable("shape with unknown special kind had no suggested VWT");
 }
 
+const uint8_t *
+ExtendedExistentialTypeCacheEntry::getOrCreateTypeLayout(Key key) {
```

Comment:
This is okay because we don't access the type layout string today in this case?

Reply:
Yes, I changed it to `nullptr`.